Service/Sources/Domain/LectureDomain/DataSource/RemoteLectureDataSource.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 public protocol RemoteLectureDataSource: BaseRemoteDataSource<LectureAPI> {
     func lectureOpen(req: LectureOpenRequestDTO) async throws
-    func lectureListInquiry() async throws
+    func lectureListInquiry() async throws -> [LectureListEntity]
     func lectureDetailInquiry(userID: String) async throws
```

Comment:
```suggestion
    func queryLectureList() async throws -> [LectureListEntity]
    func queryLectureDetail(userID: String) async throws
```
다른 조회 함수랑 같이 Inquiry말고 Query로 바꾸는 거 어떤가요??